deploy/development/deployment.tpl

```diff
@@ -91,7 +91,7 @@ spec:
             - name: BASIC_AUTH
               valueFrom:
                 secretKeyRef:
-                  name: intranet-basic-auth
```

Comment:
If there's any reference to this in the codebase it will need to be changed. I think I've seen reference to it in ingress on an nginx config. Maybe also worth checking cloud-platform-environments / intranet-dev.

Reply:
I've reverted this since I'm just using the username and password in the php